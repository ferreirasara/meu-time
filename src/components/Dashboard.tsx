import { useState } from "react";
import { FiltersCard } from "./FiltersCard";
import { PlayersTable } from './PlayersTable';
import { GeneralStats } from './GeneralStats';
import { Col, Empty, Layout, Row, theme } from 'antd';
import { Content } from 'antd/es/layout/layout';
import { AppHeader } from './AppHeader';

export const Dashboard = () => {
  const {
    token: { colorBgContainer },
  } = theme.useToken();

  const [selectedCountry, setSelectedCountry] = useState<string>("");
  const [selectedLeague, setSelectedLeague] = useState<number>(0);
  const [selectedSeason, setSelectedSeason] = useState<number>(0);
  const [selectedTeam, setSelectedTeam] = useState<number>(0);

  const handleSelectCountry = (country: string) => setSelectedCountry(country);
  const handleSelectLeague = (league: number) => setSelectedLeague(league);
  const handleSelectSeason = (handleSelectSeason: number) => setSelectedSeason(handleSelectSeason);
  const handleSelectTeam = (handleSelectTeam: number) => setSelectedTeam(handleSelectTeam);

  const showStats = !!selectedCountry && !!selectedLeague && !!selectedSeason && !!selectedTeam;

  return <Layout className="layout">
    <AppHeader />
    <Content style={{ padding: '16px 36px', background: colorBgContainer, display: 'flex', justifyContent: 'center' }}>
      <div style={{ maxWidth: 1300 }}>
        <Row>
          <Col flex="auto">
            <FiltersCard
              selectedCountry={selectedCountry}
              handleSelectCountry={handleSelectCountry}
              selectedLeague={selectedLeague}
              handleSelectLeague={handleSelectLeague}
              selectedSeason={selectedSeason}
              handleSelectSeason={handleSelectSeason}
              selectedTeam={selectedTeam}
              handleSelectTeam={handleSelectTeam}
            />
          </Col>
        </Row>
        {showStats
          ? <div>
            <GeneralStats
              selectedLeague={selectedLeague}
              selectedSeason={selectedSeason}
              selectedTeam={selectedTeam}
            />
            <Row gutter={[16, 16]}>
              <Col flex="auto">
                <PlayersTable
                  selectedLeague={selectedLeague}
                  selectedSeason={selectedSeason}
                  selectedTeam={selectedTeam}
                />
              </Col>
            </Row>
          </div>
          : <Empty
            style={{ margin: '16px' }}
            description="Selecione um time utilizando os filtros acima."
            image={Empty.PRESENTED_IMAGE_SIMPLE}
          />
        }
      </div>
    </Content>
  </Layout>
}