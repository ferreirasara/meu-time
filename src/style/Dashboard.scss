.dashboard-container {
  display: flex;
  justify-content: flex-start;
  align-items: center;
  flex-direction: column;
  gap: 16px;
  padding: 16px;
}

.stats-container {
  display: flex;
  flex-wrap: wrap;
  gap: 16px;
  max-width: 900px;
}

.chart-container {
  max-width: 900px;
}