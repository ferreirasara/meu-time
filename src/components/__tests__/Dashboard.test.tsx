import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import { Dashboard } from '../Dashboard';

describe('Dashboard', () => {
  beforeAll(() => {
    window.matchMedia = (query) => ({
      matches: false,
      media: query,
      onchange: null,
      addListener: jest.fn(), // deprecated
      removeListener: jest.fn(), // deprecated
      addEventListener: jest.fn(),
      removeEventListener: jest.fn(),
      dispatchEvent: jest.fn(),
    })
  })

  it('match snapshot', () => {
    const dashboard = renderer.create(<Dashboard />).toJSON();
    expect(dashboard).toMatchSnapshot();
  });

  it('renders empty text', () => {
    const { getByText } = render(<Dashboard />);

    const emptyEl = getByText(/Selecione um time utilizando os filtros acima./);
    expect(emptyEl).toBeInTheDocument();
  });
})