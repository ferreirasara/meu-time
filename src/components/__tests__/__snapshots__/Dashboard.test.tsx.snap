// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard match snapshot 1`] = `
<section
  className="ant-layout layout css-dev-only-do-not-override-15rg2km"
>
  <header
    className="ant-layout-header css-dev-only-do-not-override-15rg2km"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "space-between",
      }
    }
  >
    <h2
      className="ant-typography css-dev-only-do-not-override-15rg2km"
      style={
        Object {
          "WebkitLineClamp": undefined,
          "color": "#ffffff",
          "margin": 0,
        }
      }
    >
      <span
        aria-label="bar-chart"
        className="anticon anticon-bar-chart"
        role="img"
      >
        <svg
          aria-hidden="true"
          data-icon="bar-chart"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M888 792H200V168c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v688c0 4.4 3.6 8 8 8h752c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8zm-600-80h56c4.4 0 8-3.6 8-8V560c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v144c0 4.4 3.6 8 8 8zm152 0h56c4.4 0 8-3.6 8-8V384c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v320c0 4.4 3.6 8 8 8zm152 0h56c4.4 0 8-3.6 8-8V462c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v242c0 4.4 3.6 8 8 8zm152 0h56c4.4 0 8-3.6 8-8V304c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v400c0 4.4 3.6 8 8 8z"
          />
        </svg>
      </span>
       Meu Time
    </h2>
    <div
      className="ant-space css-dev-only-do-not-override-15rg2km ant-space-horizontal ant-space-align-center"
      style={Object {}}
    >
      <div
        className="ant-space-item"
        style={
          Object {
            "marginRight": 8,
          }
        }
      >
        <button
          className="ant-btn css-dev-only-do-not-override-15rg2km ant-btn-default ant-btn-icon-only ant-btn-background-ghost"
          data-testid="info-button"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          <span
            className="ant-btn-icon"
          >
            <span
              aria-label="info"
              className="anticon anticon-info"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="info"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M448 224a64 64 0 10128 0 64 64 0 10-128 0zm96 168h-64c-4.4 0-8 3.6-8 8v464c0 4.4 3.6 8 8 8h64c4.4 0 8-3.6 8-8V400c0-4.4-3.6-8-8-8z"
                />
              </svg>
            </span>
          </span>
        </button>
      </div>
      <div
        className="ant-space-item"
        style={Object {}}
      >
        <button
          className="ant-btn css-dev-only-do-not-override-15rg2km ant-btn-default ant-btn-background-ghost"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          <span
            className="ant-btn-icon"
          >
            <span
              aria-label="logout"
              className="anticon anticon-logout"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="logout"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M868 732h-70.3c-4.8 0-9.3 2.1-12.3 5.8-7 8.5-14.5 16.7-22.4 24.5a353.84 353.84 0 01-112.7 75.9A352.8 352.8 0 01512.4 866c-47.9 0-94.3-9.4-137.9-27.8a353.84 353.84 0 01-112.7-75.9 353.28 353.28 0 01-76-112.5C167.3 606.2 158 559.9 158 512s9.4-94.2 27.8-137.8c17.8-42.1 43.4-80 76-112.5s70.5-58.1 112.7-75.9c43.6-18.4 90-27.8 137.9-27.8 47.9 0 94.3 9.3 137.9 27.8 42.2 17.8 80.1 43.4 112.7 75.9 7.9 7.9 15.3 16.1 22.4 24.5 3 3.7 7.6 5.8 12.3 5.8H868c6.3 0 10.2-7 6.7-12.3C798 160.5 663.8 81.6 511.3 82 271.7 82.6 79.6 277.1 82 516.4 84.4 751.9 276.2 942 512.4 942c152.1 0 285.7-78.8 362.3-197.7 3.4-5.3-.4-12.3-6.7-12.3zm88.9-226.3L815 393.7c-5.3-4.2-13-.4-13 6.3v76H488c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h314v76c0 6.7 7.8 10.5 13 6.3l141.9-112a8 8 0 000-12.6z"
                />
              </svg>
            </span>
          </span>
          <span>
            Sair
          </span>
        </button>
      </div>
    </div>
  </header>
  <main
    className="ant-layout-content css-dev-only-do-not-override-15rg2km"
    style={
      Object {
        "background": "#ffffff",
        "display": "flex",
        "justifyContent": "center",
        "padding": "16px 36px",
      }
    }
  >
    <div
      style={
        Object {
          "maxWidth": 1300,
        }
      }
    >
      <div
        className="ant-space css-dev-only-do-not-override-15rg2km ant-space-vertical"
        style={Object {}}
      >
        <div
          className="ant-space-item"
          style={
            Object {
              "marginBottom": 8,
            }
          }
        >
          <div
            className="ant-row css-dev-only-do-not-override-15rg2km"
            style={Object {}}
          >
            <div
              className="ant-col css-dev-only-do-not-override-15rg2km"
              style={
                Object {
                  "flex": "auto",
                }
              }
            >
              <div
                className="ant-card ant-card-bordered css-dev-only-do-not-override-15rg2km"
                style={
                  Object {
                    "maxWidth": 1300,
                  }
                }
              >
                <div
                  className="ant-card-body"
                  style={Object {}}
                >
                  <form
                    className="ant-form ant-form-inline css-dev-only-do-not-override-15rg2km"
                    onReset={[Function]}
                    onSubmit={[Function]}
                  >
                    <div
                      className="ant-form-item css-dev-only-do-not-override-15rg2km"
                    >
                      <div
                        className="ant-row ant-form-item-row css-dev-only-do-not-override-15rg2km"
                        style={Object {}}
                      >
                        <div
                          className="ant-col ant-form-item-label css-dev-only-do-not-override-15rg2km"
                          style={Object {}}
                        >
                          <label
                            className=""
                            htmlFor="country"
                            title="País"
                          >
                            País
                          </label>
                        </div>
                        <div
                          className="ant-col ant-form-item-control css-dev-only-do-not-override-15rg2km"
                          style={Object {}}
                        >
                          <div
                            className="ant-form-item-control-input"
                          >
                            <div
                              className="ant-form-item-control-input-content"
                            >
                              <div
                                className="ant-select ant-select-in-form-item css-dev-only-do-not-override-15rg2km ant-select-single ant-select-show-arrow ant-select-disabled ant-select-loading ant-select-show-search"
                                data-testid="select-country"
                                onBlur={[Function]}
                                onFocus={[Function]}
                                onKeyDown={[Function]}
                                onKeyUp={[Function]}
                                onMouseDown={[Function]}
                                style={
                                  Object {
                                    "width": 150,
                                  }
                                }
                              >
                                <div
                                  className="ant-select-selector"
                                  onClick={[Function]}
                                  onMouseDown={[Function]}
                                >
                                  <span
                                    className="ant-select-selection-search"
                                  >
                                    <input
                                      aria-activedescendant="country_list_0"
                                      aria-autocomplete="list"
                                      aria-controls="country_list"
                                      aria-expanded={false}
                                      aria-haspopup="listbox"
                                      aria-owns="country_list"
                                      autoComplete="off"
                                      className="ant-select-selection-search-input"
                                      disabled={true}
                                      id="country"
                                      onChange={[Function]}
                                      onCompositionEnd={[Function]}
                                      onCompositionStart={[Function]}
                                      onKeyDown={[Function]}
                                      onMouseDown={[Function]}
                                      onPaste={[Function]}
                                      readOnly={false}
                                      role="combobox"
                                      style={
                                        Object {
                                          "opacity": null,
                                        }
                                      }
                                      type="search"
                                      unselectable={null}
                                      value=""
                                    />
                                  </span>
                                  <span
                                    className="ant-select-selection-placeholder"
                                  >
                                    Selecione um país
                                  </span>
                                </div>
                                <span
                                  aria-hidden={true}
                                  className="ant-select-arrow ant-select-arrow-loading"
                                  onMouseDown={[Function]}
                                  style={
                                    Object {
                                      "WebkitUserSelect": "none",
                                      "userSelect": "none",
                                    }
                                  }
                                  unselectable="on"
                                >
                                  <span
                                    aria-label="loading"
                                    className="anticon anticon-loading anticon-spin"
                                    role="img"
                                  >
                                    <svg
                                      aria-hidden="true"
                                      data-icon="loading"
                                      fill="currentColor"
                                      focusable="false"
                                      height="1em"
                                      viewBox="0 0 1024 1024"
                                      width="1em"
                                    >
                                      <path
                                        d="M988 548c-19.9 0-36-16.1-36-36 0-59.4-11.6-117-34.6-171.3a440.45 440.45 0 00-94.3-139.9 437.71 437.71 0 00-139.9-94.3C629 83.6 571.4 72 512 72c-19.9 0-36-16.1-36-36s16.1-36 36-36c69.1 0 136.2 13.5 199.3 40.3C772.3 66 827 103 874 150c47 47 83.9 101.8 109.7 162.7 26.7 63.1 40.2 130.2 40.2 199.3.1 19.9-16 36-35.9 36z"
                                      />
                                    </svg>
                                  </span>
                                </span>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                    <div
                      className="ant-form-item css-dev-only-do-not-override-15rg2km"
                    >
                      <div
                        className="ant-row ant-form-item-row css-dev-only-do-not-override-15rg2km"
                        style={Object {}}
                      >
                        <div
                          className="ant-col ant-form-item-label css-dev-only-do-not-override-15rg2km"
                          style={Object {}}
                        >
                          <label
                            className=""
                            htmlFor="league"
                            title="Liga"
                          >
                            Liga
                          </label>
                        </div>
                        <div
                          className="ant-col ant-form-item-control css-dev-only-do-not-override-15rg2km"
                          style={Object {}}
                        >
                          <div
                            className="ant-form-item-control-input"
                          >
                            <div
                              className="ant-form-item-control-input-content"
                            >
                              <div
                                className="ant-select ant-select-in-form-item css-dev-only-do-not-override-15rg2km ant-select-single ant-select-show-arrow ant-select-disabled ant-select-loading ant-select-show-search"
                                data-testid="select-league"
                                onBlur={[Function]}
                                onFocus={[Function]}
                                onKeyDown={[Function]}
                                onKeyUp={[Function]}
                                onMouseDown={[Function]}
                                style={
                                  Object {
                                    "width": 150,
                                  }
                                }
                              >
                                <div
                                  className="ant-select-selector"
                                  onClick={[Function]}
                                  onMouseDown={[Function]}
                                >
                                  <span
                                    className="ant-select-selection-search"
                                  >
                                    <input
                                      aria-activedescendant="league_list_0"
                                      aria-autocomplete="list"
                                      aria-controls="league_list"
                                      aria-expanded={false}
                                      aria-haspopup="listbox"
                                      aria-owns="league_list"
                                      autoComplete="off"
                                      className="ant-select-selection-search-input"
                                      disabled={true}
                                      id="league"
                                      onChange={[Function]}
                                      onCompositionEnd={[Function]}
                                      onCompositionStart={[Function]}
                                      onKeyDown={[Function]}
                                      onMouseDown={[Function]}
                                      onPaste={[Function]}
                                      readOnly={false}
                                      role="combobox"
                                      style={
                                        Object {
                                          "opacity": null,
                                        }
                                      }
                                      type="search"
                                      unselectable={null}
                                      value=""
                                    />
                                  </span>
                                  <span
                                    className="ant-select-selection-placeholder"
                                  >
                                    Selecione uma liga
                                  </span>
                                </div>
                                <span
                                  aria-hidden={true}
                                  className="ant-select-arrow ant-select-arrow-loading"
                                  onMouseDown={[Function]}
                                  style={
                                    Object {
                                      "WebkitUserSelect": "none",
                                      "userSelect": "none",
                                    }
                                  }
                                  unselectable="on"
                                >
                                  <span
                                    aria-label="loading"
                                    className="anticon anticon-loading anticon-spin"
                                    role="img"
                                  >
                                    <svg
                                      aria-hidden="true"
                                      data-icon="loading"
                                      fill="currentColor"
                                      focusable="false"
                                      height="1em"
                                      viewBox="0 0 1024 1024"
                                      width="1em"
                                    >
                                      <path
                                        d="M988 548c-19.9 0-36-16.1-36-36 0-59.4-11.6-117-34.6-171.3a440.45 440.45 0 00-94.3-139.9 437.71 437.71 0 00-139.9-94.3C629 83.6 571.4 72 512 72c-19.9 0-36-16.1-36-36s16.1-36 36-36c69.1 0 136.2 13.5 199.3 40.3C772.3 66 827 103 874 150c47 47 83.9 101.8 109.7 162.7 26.7 63.1 40.2 130.2 40.2 199.3.1 19.9-16 36-35.9 36z"
                                      />
                                    </svg>
                                  </span>
                                </span>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                    <div
                      className="ant-form-item css-dev-only-do-not-override-15rg2km"
                    >
                      <div
                        className="ant-row ant-form-item-row css-dev-only-do-not-override-15rg2km"
                        style={Object {}}
                      >
                        <div
                          className="ant-col ant-form-item-label css-dev-only-do-not-override-15rg2km"
                          style={Object {}}
                        >
                          <label
                            className=""
                            htmlFor="season"
                            title="Temporada"
                          >
                            Temporada
                          </label>
                        </div>
                        <div
                          className="ant-col ant-form-item-control css-dev-only-do-not-override-15rg2km"
                          style={Object {}}
                        >
                          <div
                            className="ant-form-item-control-input"
                          >
                            <div
                              className="ant-form-item-control-input-content"
                            >
                              <div
                                className="ant-select ant-select-in-form-item css-dev-only-do-not-override-15rg2km ant-select-single ant-select-show-arrow ant-select-disabled ant-select-loading ant-select-show-search"
                                data-testid="select-season"
                                onBlur={[Function]}
                                onFocus={[Function]}
                                onKeyDown={[Function]}
                                onKeyUp={[Function]}
                                onMouseDown={[Function]}
                                style={
                                  Object {
                                    "width": 150,
                                  }
                                }
                              >
                                <div
                                  className="ant-select-selector"
                                  onClick={[Function]}
                                  onMouseDown={[Function]}
                                >
                                  <span
                                    className="ant-select-selection-search"
                                  >
                                    <input
                                      aria-activedescendant="season_list_0"
                                      aria-autocomplete="list"
                                      aria-controls="season_list"
                                      aria-expanded={false}
                                      aria-haspopup="listbox"
                                      aria-owns="season_list"
                                      autoComplete="off"
                                      className="ant-select-selection-search-input"
                                      disabled={true}
                                      id="season"
                                      onChange={[Function]}
                                      onCompositionEnd={[Function]}
                                      onCompositionStart={[Function]}
                                      onKeyDown={[Function]}
                                      onMouseDown={[Function]}
                                      onPaste={[Function]}
                                      readOnly={false}
                                      role="combobox"
                                      style={
                                        Object {
                                          "opacity": null,
                                        }
                                      }
                                      type="search"
                                      unselectable={null}
                                      value=""
                                    />
                                  </span>
                                  <span
                                    className="ant-select-selection-placeholder"
                                  >
                                    Selecione uma temporada
                                  </span>
                                </div>
                                <span
                                  aria-hidden={true}
                                  className="ant-select-arrow ant-select-arrow-loading"
                                  onMouseDown={[Function]}
                                  style={
                                    Object {
                                      "WebkitUserSelect": "none",
                                      "userSelect": "none",
                                    }
                                  }
                                  unselectable="on"
                                >
                                  <span
                                    aria-label="loading"
                                    className="anticon anticon-loading anticon-spin"
                                    role="img"
                                  >
                                    <svg
                                      aria-hidden="true"
                                      data-icon="loading"
                                      fill="currentColor"
                                      focusable="false"
                                      height="1em"
                                      viewBox="0 0 1024 1024"
                                      width="1em"
                                    >
                                      <path
                                        d="M988 548c-19.9 0-36-16.1-36-36 0-59.4-11.6-117-34.6-171.3a440.45 440.45 0 00-94.3-139.9 437.71 437.71 0 00-139.9-94.3C629 83.6 571.4 72 512 72c-19.9 0-36-16.1-36-36s16.1-36 36-36c69.1 0 136.2 13.5 199.3 40.3C772.3 66 827 103 874 150c47 47 83.9 101.8 109.7 162.7 26.7 63.1 40.2 130.2 40.2 199.3.1 19.9-16 36-35.9 36z"
                                      />
                                    </svg>
                                  </span>
                                </span>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                    <div
                      className="ant-form-item css-dev-only-do-not-override-15rg2km"
                    >
                      <div
                        className="ant-row ant-form-item-row css-dev-only-do-not-override-15rg2km"
                        style={Object {}}
                      >
                        <div
                          className="ant-col ant-form-item-label css-dev-only-do-not-override-15rg2km"
                          style={Object {}}
                        >
                          <label
                            className=""
                            htmlFor="team"
                            title="Time"
                          >
                            Time
                          </label>
                        </div>
                        <div
                          className="ant-col ant-form-item-control css-dev-only-do-not-override-15rg2km"
                          style={Object {}}
                        >
                          <div
                            className="ant-form-item-control-input"
                          >
                            <div
                              className="ant-form-item-control-input-content"
                            >
                              <div
                                className="ant-select ant-select-in-form-item css-dev-only-do-not-override-15rg2km ant-select-single ant-select-show-arrow ant-select-disabled ant-select-loading ant-select-show-search"
                                data-testid="select-team"
                                onBlur={[Function]}
                                onFocus={[Function]}
                                onKeyDown={[Function]}
                                onKeyUp={[Function]}
                                onMouseDown={[Function]}
                                style={
                                  Object {
                                    "width": 150,
                                  }
                                }
                              >
                                <div
                                  className="ant-select-selector"
                                  onClick={[Function]}
                                  onMouseDown={[Function]}
                                >
                                  <span
                                    className="ant-select-selection-search"
                                  >
                                    <input
                                      aria-activedescendant="team_list_0"
                                      aria-autocomplete="list"
                                      aria-controls="team_list"
                                      aria-expanded={false}
                                      aria-haspopup="listbox"
                                      aria-owns="team_list"
                                      autoComplete="off"
                                      className="ant-select-selection-search-input"
                                      disabled={true}
                                      id="team"
                                      onChange={[Function]}
                                      onCompositionEnd={[Function]}
                                      onCompositionStart={[Function]}
                                      onKeyDown={[Function]}
                                      onMouseDown={[Function]}
                                      onPaste={[Function]}
                                      readOnly={false}
                                      role="combobox"
                                      style={
                                        Object {
                                          "opacity": null,
                                        }
                                      }
                                      type="search"
                                      unselectable={null}
                                      value=""
                                    />
                                  </span>
                                  <span
                                    className="ant-select-selection-placeholder"
                                  >
                                    Selecione um time
                                  </span>
                                </div>
                                <span
                                  aria-hidden={true}
                                  className="ant-select-arrow ant-select-arrow-loading"
                                  onMouseDown={[Function]}
                                  style={
                                    Object {
                                      "WebkitUserSelect": "none",
                                      "userSelect": "none",
                                    }
                                  }
                                  unselectable="on"
                                >
                                  <span
                                    aria-label="loading"
                                    className="anticon anticon-loading anticon-spin"
                                    role="img"
                                  >
                                    <svg
                                      aria-hidden="true"
                                      data-icon="loading"
                                      fill="currentColor"
                                      focusable="false"
                                      height="1em"
                                      viewBox="0 0 1024 1024"
                                      width="1em"
                                    >
                                      <path
                                        d="M988 548c-19.9 0-36-16.1-36-36 0-59.4-11.6-117-34.6-171.3a440.45 440.45 0 00-94.3-139.9 437.71 437.71 0 00-139.9-94.3C629 83.6 571.4 72 512 72c-19.9 0-36-16.1-36-36s16.1-36 36-36c69.1 0 136.2 13.5 199.3 40.3C772.3 66 827 103 874 150c47 47 83.9 101.8 109.7 162.7 26.7 63.1 40.2 130.2 40.2 199.3.1 19.9-16 36-35.9 36z"
                                      />
                                    </svg>
                                  </span>
                                </span>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </form>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="ant-space-item"
          style={Object {}}
        >
          <div
            className="css-dev-only-do-not-override-15rg2km ant-empty ant-empty-normal"
            style={
              Object {
                "margin": "16px",
              }
            }
          >
            <div
              className="ant-empty-image"
            >
              <svg
                height="41"
                viewBox="0 0 64 41"
                width="64"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="none"
                  fillRule="evenodd"
                  transform="translate(0 1)"
                >
                  <ellipse
                    cx="32"
                    cy="33"
                    fill="#f5f5f5"
                    rx="32"
                    ry="7"
                  />
                  <g
                    fillRule="nonzero"
                    stroke="#d9d9d9"
                  >
                    <path
                      d="M55 12.76L44.854 1.258C44.367.474 43.656 0 42.907 0H21.093c-.749 0-1.46.474-1.947 1.257L9 12.761V22h46v-9.24z"
                    />
                    <path
                      d="M41.613 15.931c0-1.605.994-2.93 2.227-2.931H55v18.137C55 33.26 53.68 35 52.05 35h-40.1C10.32 35 9 33.259 9 31.137V13h11.16c1.233 0 2.227 1.323 2.227 2.928v.022c0 1.605 1.005 2.901 2.237 2.901h14.752c1.232 0 2.237-1.308 2.237-2.913v-.007z"
                      fill="#fafafa"
                    />
                  </g>
                </g>
              </svg>
            </div>
            <div
              className="ant-empty-description"
            >
              Selecione um time utilizando os filtros acima.
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>
</section>
`;
